zipkin4net-aspnetcore/Criteo.Profiling.Tracing.Middleware/TracingMiddleware.cs

```diff
@@ -19,19 +19,8 @@ public static void UseTracing(this IApplicationBuilder app, string serviceName)
                 trace.Record(Annotations.ServerRecv());
                 trace.Record(Annotations.ServiceName(serviceName));
                 trace.Record(Annotations.Rpc(context.Request.Method));
-                try
-                {
-                    await next.Invoke();
-                }
-                catch (System.Exception e)
-                {
-                    trace.Record(Annotations.Tag("error", e.Message));
-                    throw;
-                }
-                finally
-                {
-                    trace.Record(Annotations.ServerSend());
-                }
+                await next.Invoke();
```

Comment:
Actually, this is being refactored in https://github.com/criteo/zipkin4net/pull/31